backend/bittan/bittan/tests/views/test_views.py

```diff
@@ -58,7 +58,7 @@ def test_filled(self):
 				raise Exception("No corresponding ticket_type found")
 			self.assertEqual(tt['title'], corresponding_tt_db.title)
 			self.assertEqual(tt['description'], corresponding_tt_db.description)
-			self.assertEqual(type(tt['price']), str) # We don't bother checking Money too closely because we're gonna change it anyway
+			self.assertEqual(type(tt['price']), int) # We don't bother checking Money too closely because we're gonna change it anyway

```

Comment:
Remove comment, it is no longer true! Also, we can now check that exact value of price.

Reply:
Solved in d8f85f9f48cb.